pkg/probe/vpn_ssl_test.go

```diff
@@ -12,6 +14,8 @@ func TestVPNSsl(t *testing.T) {
 	c := newFakeClient()
 	c.prepare("api/v2/monitor/vpn/ssl", "testdata/vpn.jsonnet")
 	r := prometheus.NewPedanticRegistry()
+	flag.Set("max-vpn-users", "10")
```

Comment:
```suggestion
	flag.Set("max-vpn-users", "0")
```

Should be 0, if reading the discussions.

Reply:
Ignore this I have overlooked that this is a test